core/src/avm1/globals/array.rs

```diff
@@ -674,70 +674,61 @@ fn qsort<'gc>(
         return Ok(());
     }
 
-    // Fast-path for 2 elements.
```

Comment:
To be sure, did the fast-path make no difference in perf in your case?

Reply:
> did the fast-path make no difference in perf in your case?

I didn’t specifically test a fast-path for two elements, but I kept it from the previous implementation

Reply:
I'm confused, you said you kept the fast path, but I don't see it in the diff.
(I'm not asking it to be added or not, just clarifying)

Reply:
> I don't see it in the diff.

Sorry, you're right. I thought it wouldn't make much difference in an iterative algorithm.